### PR TITLE
ECF-RP-411: Map routes to apps

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
We have domains, we need to link them up to the apps for staging and prod

### Changes proposed in this pull request
Draw inspiration from teaching vacancies

### Testing
ran a `terraform plan` for staging locally and it looked good

### CODE REVIEW

**Comment by ethanmills** [comment]
I've also bundled in a couple of other small changes, passing through the SHA to the app properly and putting the turnstyle in the right place

File: .github/workflows/deploy_to_dev.yml [hunk 1]
```diff
@@ -8,9 +8,8 @@ on:
       - 'documentation/**'
 
 jobs:
-  turnstyle:
```
Comment (kicferk1) [inline .github/workflows/deploy_to_dev.yml:11]: do we hate turnstyle? I know we copied it from somewhere else, is it not doing anything useful?

**Review by kicferk1 — APPROVED** [review]
Looks good to me

File: .github/workflows/deploy_to_dev.yml (hunk 1 above)
Comment (kicferk1) [reply .github/workflows/deploy_to_dev.yml:11]: Oh, sorry, I misread the changes, we still use it only elsewhere. Git diffs can be confusing.

**Pull request merged**